README.md

```diff
@@ -4,28 +4,5 @@ Projekty wykonywane przez społeczność discorda
 Projekt jest we wczesnej fazie i obecne challenge sa testowe. Wszelkie sugestie propozycje pomocy mile widziane.
 
 ## Challenge #1 - Landing Page
-Celem wyzwania jest wykonanie warstwy frontendowej Landing Page, zbierającego adressy e-mail osób zainteresowanych udziałem w akademii.  
-Stack Technologiczny do wykonania zadania to html/css/js bez zadnych frameworkow, preprocesorow.  
-Rozwiązanie powinno dzialać na: 
-* Chrome 58+
-* Edge  16+
-* Safari 10.1+
-* Firefox 54+
-Przycisk do wysyania formularza powinien mieć ID: LandingEmail  
-Wykonane zadanie należy przesylac jako pull request na branch develop.  
-Zwrocie uwagę na sematyke html i accessibility.  
 
-### Grafiki
-[Project.pdf](/assets/Layout.pdf.pdf})  
-[Figma](https://www.figma.com/file/gCTf2Ux96ETR0DMxr6T2m039/LandingPage?node-id=1%3A13)  
-[Zeppelin](https://zpl.io/boKzeQv)  
-<sub> Marginesy mogą się trochę nie zgadzać dopiero się uczę figmy</sub>
-#### Bonus challenge 
-* Wykonaj zadanie bez użycia javascriptu
-* Stwórz warstwe backendowa(dowolny język) lub podepnij się do firebase
-### Termin 
-Wstępny czas na wykonanie challenga 21.01.2019  
-  
-  
-
-Jak ktoś ma ochotę zrobić lepszy readme, to niech to zrobi :D
+demo - http://vizman.ayz.pl/01-Landing-Page/
```

Comment:
Cały README usunąłeś... Unikaj tego, mało ko to lubi

Reply:
![obraz](https://user-images.githubusercontent.com/29540719/51563198-fdde4300-1e8b-11e9-8195-014fa909fc1c.png)
Musisz poprawić ;)

Reply:
Poprawione, jednak dalej chyba jest konflikt? Czyli nic do readme nie mogę dodać, nawet linka do wersji podglądowej?